Sparkle/SUBasicUpdateDriver.m

```diff
@@ -191,6 +191,11 @@ - (void)downloadUpdate
 {
     NSMutableURLRequest *request = [NSMutableURLRequest requestWithURL:[self.updateItem fileURL]];
     [request setValue:[self.updater userAgentString] forHTTPHeaderField:@"User-Agent"];
+    if ([[self.updater delegate] respondsToSelector:@selector(updater:willDownloadUpdate:withRequest:)]) {
+        [[self.updater delegate] updater:self.updater
```

Comment:
Is this using `[bracket notation]` instead of `dot.notation` to match the rest of the code?